File: _wpf/wpf-business-programmer/2-wpf-business-programmer-m2-exercise-files/Samples/VB/WPFListBox-VB/WPFComponents/MyCustomer.vb
Public Class MyCustomer
  Private mFirstName As String
  Private mLastName As String

  Public Property FirstName() As String
    Get
      Return (mFirstName)
    End Get
    Set(ByVal value As String)
      mFirstName = value
    End Set
  End Property

  Public Property LastName() As String
    Get
      Return (mLastName)
    End Get
    Set(ByVal value As String)
      mLastName = value
    End Set
  End Property

  Public ReadOnly Property FullName() As String
    Get
      Return (FirstName & " ") + LastName
    End Get
  End Property
End Class

File: _wpf/wpf-business-programmer/2-wpf-business-programmer-m2-exercise-files/Samples/CSharp/WPFListBox/WPFComponents/MyCustomer.cs
using System;

namespace WPFComponents
{
  public class MyCustomer
  {
    public string FirstName { get; set; }
    public string LastName { get; set; }
    public string FullName
    {
      get { return FirstName + " " + LastName; }
    }
  }
}
